test/IntegrationTests/LogTests.cs

```diff
@@ -52,13 +53,17 @@ public async Task SubmitLogs(bool enableClrProfiler, bool includeFormattedMessag
             collector.Expect(logRecord => Convert.ToString(logRecord).Contains("TestApplication.Logs.Controllers.TestController"));
         }
 
-        if (!enableClrProfiler)
+        if (enableClrProfiler)
+        {
+            SetEnvironmentVariable("CORECLR_ENABLE_PROFILING", "1");
```

Comment:
Since this is something that is done in multiple tests, we can probably move this to its own method so that we don't have that environment variable name copied everywhere. I think that this can be done in a separate PR.